lib/cli/scripts/plugins/aps-plugin.ts

```diff
@@ -0,0 +1,61 @@
+import { PluginInterface } from './plugin-model';
+import { logger } from '../logger';
+import { PluginConfiguration } from './plugin-config';
+
+export class ProcessServicePlugin {
+    config: PluginConfiguration;
+
+    constructor(private plugInInfo: PluginInterface, private alfrescoJsApi: any) {
+        this.config = new PluginConfiguration(this.plugInInfo, this.alfrescoJsApi, false);
```

Comment:
a method should never have a boolean. 
if you have a boolean means that you have 2 behaviour... so you can create 2 classes.

Reply:
Yup, Updated